assets/styles/components/atoms/SfBadge.scss

```diff
@@ -28,6 +28,22 @@
     font-size: var(--spacer-xs);
   }
 
+  &--number-mobile {
+    --badge-min-width: 0.875rem;
+    --badge-min-height: 0.875rem;
```

Comment:
for all height and width use spacer instead of rem

Reply:
Will resolve in next PR.